src/llms/base.test.ts

```diff
@@ -67,9 +70,13 @@ describe("BaseLLM", () => {
       return { tokenLimit: 4096 };
     }
 
+    async embed(_input: string[], _options?: EmbeddingOptions): Promise<EmbeddingOutput> {
```

Comment:
Don't use `_` here. (update it everywhere, please).

Reply:
The solution here would be to use a lint command like so?: 
```
// eslint-disable-next-line unused-imports/no-unused-vars
async embed(input: string[], options?: EmbeddingOptions): Promise<EmbeddingOutput> { ...
```

Reply:
updated https://github.com/i-am-bee/bee-agent-framework/pull/210/commits/28430045c7b17d3bf8400d69af42ce9e93d4b1a1